lib/fluent/plugin/exception_detector.rb

```diff
@@ -59,7 +59,7 @@ def self.supported
     ].freeze
 
     PYTHON_RULES = [
-      rule(:start_state, /Traceback \(most recent call last\)/, :python),
+      rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
```

Comment:
Hmm... I wonder if this is considered a breaking change. Technically the change can fix the nested json issue, while breaking certain existing customers' integration. The chance is really low though if we expect all customers to do standard Python exception trace.